src/Pact/Native.hs

```diff
@@ -286,6 +286,33 @@ namespaceDef = setTopLevelOnly $ defRNative "namespace" namespace
         Nothing  -> evalError info $
           "namespace: '" <> pretty name <> "' not defined"
 
+chainDataDef :: NativeDef
+chainDataDef = defRNative "chain-data" chainData (funType (tTyObject a) [])
+    ["(chain-data)"]
+    "Read transaction public metadata."
```

Comment:
Document the output in the docs, see https://github.com/kadena-io/pact/blob/caf1b25bcf02bb3c3ac2d83a561c608ef97356b1/src/Pact/Native/Db.hs#L136

Reply:
Updated